modules/olm-catalogsource.adoc

```diff
@@ -26,10 +26,15 @@ metadata:
  namespace: olm
 spec:
  sourceType: grpc
- image: quay.io/operator-framework/upstream-community-operators:latest
+ image: quay.io/operator-framework/upstream-community-operators:latest <1>
  displayName: Community Operators
  publisher: OperatorHub.io
```

Comment:
It's better to add the `priority: -400` here. It's a new feature for 4.6, details: https://issues.redhat.com/browse/OLM-1920
Here is the full configure for the Community Operators CatalogSource in 4.6:
```console
spec:
  displayName: Community Operators
  image: registry.redhat.io/redhat/community-operator-index:latest
  priority: -400
  publisher: Red Hat
  sourceType: grpc
  updateStrategy:
    registryPoll:
      interval: 10m0s
```